### PR TITLE
Fresh checkpoints

### LINKED ISSUE #323: generate checkpoints
- [ ] Generate the checkpoints using getTreeState from lightwalletd with 10000 block intervals since the last update for **Testnet**
- [ ] Generate the checkpoints using getTreeState from lightwalletd with 10000 block intervals since the last update for **Mainnet**

### PR DESCRIPTION
**Pull request by pacu**
closes #323

### CODE REVIEW

File: ZcashLightClientKit/Constants/WalletBirthday+mainnet.swift [hunk 1]
```diff
@@ -479,19 +479,117 @@ extension WalletBirthday {
...
+                height: 1340000,
+                hash: "00000000031bc547da975ebd77d9113b178053f88fb6a1d8511b4f8962c21c4b",
+                time: 1627846248,
+                tree: "01ef55e131bf5e7d6737a6e353fe0ff246ba8938a264335457452db2c4023241590113f4e2a1f043d0a303c769d9aac5eeb8b6854d1a64d71b6b86cda2e0eeee07621301206a8d77952d4143cc5ba4d7943261e7145f0f138a81fe37c10e50a487487966012fb54cf3a70cccf01479fefc42e539c92a8215aead4179278cf1e8a302cb4868014574313eb9fd9ee592346fdf27752f698c1f629b044437853972e266e95b56020001be3f0fa5b20bbfa445293d588073dc27a856c92e9903831c6de4455f03d57a0401bb534b0af17c990f836204115aa17d4c2504fa0a675353ec7ae8a7d67510cc46012e2edeb7e5acb0d440dd5b500bec4a6efd6f53ba02c10e3883e23e53d7f91369000183c334e455aeeeb82cceddbe832919324d7011418749fc9dea759cfa6c2cc21501f4a3504117d35efa15f57d5fdd19515b7fb1dd14c3b98b8a91685f0f788db330000000018846ec9170ad4e40a093cfb53162e5211d55377d8d22f826cde7783d30c1dd5f01b35fe4a943a47404f68db220c77b0573e13c3378a65c6f2396f93be7609d8f2a000125911f4524469c00ccb1ba69e64f0ee7380c8d17bbfc76ecd238421b86eb6e09000118f64df255c9c43db708255e7bf6bffd481e5c2f38fe9ed8f3d189f7f9cf2644"
+            )
+        case 1350000 ..< 1350000:
```
Comment (adamstener) [inline ZcashLightClientKit/Constants/WalletBirthday+mainnet.swift:496]: Should this be `case 1350000 ..< 1360000:` ?

Comment (pacu) [reply ZcashLightClientKit/Constants/WalletBirthday+mainnet.swift:496]: thanks! this is the kind of proof-reading I was looking for! ❤️

Comment (adamstener) [reply ZcashLightClientKit/Constants/WalletBirthday+mainnet.swift:496]: 🙌

**Review by adamstener — APPROVED** [review]
I'm not entirely sure what's going on with this one but it looks consistent to me 👍

**Comment by pacu** [comment]
Thanks for the review @adamstener.  These changes correspond to updating the checkpoints (tree states) that wallets use as starting points to sync up with the blockchain. the oldest the checkpoint, the longer it takes to restore or sync up a new wallet

**Pull request merged**